README.adoc

```diff
@@ -6,6 +6,37 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Operators
+
+In a new terminal, navigate to codeready-toolchain/host-operator directory and execute the following commands
+```
+$ minishift start --profile toolchain
+$ make up-local
+^C
+```
+In the same terminal, navigate to codeready-toolchain/member-operator directory and execute the following commands
```

Comment:
From my POV, no need to say in same_terminal and navigate etc.

Instead,
```
$ cd $GOPATH/github.com/codeready-toolchain/member-operator
$ make up-local
^c 
```
Rather explaining in sentence .. do it using `cd` command.

Reply:
@nurali-techie That sounds much more clear! Will make that change. Thanks :-)